target/linux/ramips/rt305x/base-files/etc/board.d/02_network

```diff
@@ -207,6 +208,7 @@ ramips_setup_macs()
 	omnima,miniembplug|\
 	planex,mzk-w300nh2|\
 	sitecom,wl-351|\
+	tenda,a6|\
```

Comment:
Above, you define it as a one-port. So why should it have an address for WAN defined?